src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -533,8 +533,14 @@ public Response getSiteMembershipAPI(
 			String sessionId = getUserSessionId(request);
 
 			// return CTools site members
-			return Response.status(Response.Status.OK).entity(get_site_members(siteId, sessionId))
-					.build();
+			HashMap<String, String> site_members = get_site_members(siteId, sessionId);
+			for (String userEid : site_members.keySet()) {
+				if (userEid.equals("errorMessage")) {
+						return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity(site_members).build();
+				}
+			}
+
+			return Response.status(Response.Status.OK).entity(site_members).build();
 		} catch (Exception e) {
```

Comment:
Exception is too general to catch safely.  The catch needs to be more specific.  At the very, very least the exception should be logged at error level so that it is possible to see what the error is and update the code to handle the correct exception.